bindings/python/utils/pickle-map.hpp

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright (c) 2019 CNRS
+//
+
+#ifndef __pinocchio_python_utils_pickle_map_hpp__
+#define __pinocchio_python_utils_pickle_map_hpp__
+
+#include "pinocchio/bindings/python/utils/pickle-vector.hpp"
+
+namespace pinocchio
+{
+  namespace python
+  {
+    namespace bp = boost::python;
+    ///
+    /// \brief Create a pickle interface for the std::map and aligned map
+    ///
+    /// \tparam VecType Map Type to pickle
+    ///
+    /// \sa Pickle
+    ///
+    template<typename VecType>
+    struct PickleMap : public PickleVector<VecType>
+    { 
+      static void setstate(bp::object op, bp::tuple tup)
```

Comment:
I just realized that this one is also duplicate. It would be worthy to remove it.

Reply:
std::map::insert has a slightly different API than std::vector::insert. 
vector.insert also requires the index where to insert. Other than that, it is the same.